### ISSUE TITLE
Subscription links are not descriptive

### ISSUE THREAD

**Issue by jon-kirwan** [alphagov/govuk_publishing_components#2991]
## What:

Screen reader users were unable to identify the destination or purpose of the **email** and **feed** [subscription links](https://components.publishing.service.gov.uk/component-guide/subscription_links) at the bottom of the page when navigating out of context.

**Example URL:**

https://www.gov.uk/foreign-travel-advice

## Why:

Link text should be unique and descriptive to help users identify the links destination or purpose when navigating out of context.

For example: **Get email updates for all countries** or **Get RSS feed updates for all countries**

**Comment by andysellick** [comment]
Looks like the default text of this component is much more understandable (`Get emails` and `subscribe to feed`) but has in this case [been overridden](https://github.com/alphagov/frontend/blob/main/app/views/travel_advice/index.html.erb#L65-L67). We could start by removing these overrides.

**Comment by selfthinker** [comment]
Should this instead be given to the team who owns the app that has changed the link text?

**Comment by jon-kirwan** [comment]
> Should this instead be given to the team who owns the app that has changed the link text?

I think so. From a [quick search](https://github.com/search?q=org%3Aalphagov+subscription_links+NOT+repo%3Aalphagov%2Fgovuk_publishing_components&type=code), it looks like the changes are in Collections, Frontend and Government Frontend - so perhaps should be passed to the #govuk-navigation-tech team

**Comment by selfthinker** [comment]
> so perhaps should be passed to the #govuk-navigation-tech team

Thanks. I've talked to them and they've [added this to their backlog](https://trello.com/c/d3cD6jPN).

**Comment by jon-kirwan** [comment]
Great thanks for the update. I'll close the issue.
